plugins/base/src/main/kotlin/transformers/documentables/DefaultDocumentableMerger.kt

```diff
@@ -43,7 +45,16 @@ private fun <T> mergeExpectActual(
 
     fun analyzeExpectActual(sameDriElements: List<T>) = sameDriElements.reduce(reducer)
 
-    return elements.groupBy { it.dri }.values.map(::analyzeExpectActual)
+    fun T.isExpectActual() : Boolean =
+        this.safeAs<WithExtraProperties<T>>().let { it != null && it.extra[IsExpectActual] != null }
+
+    return elements.groupBy {
+        it.isExpectActual()
+    }.let {
+        it[false].orEmpty() +
```

Comment:
I love `it[true]` and `it[false]` but others won't appreciate its genius

I'd suggest using `partition` to split the list and then `map`

Reply:
ohhh partition is what I wanted to have there, didnt know it exists... thanks